equinix/resource_network_device.go

```diff
@@ -1467,7 +1478,7 @@ func getNetworkDeviceStateChangeConfigs(c ne.Client, deviceID string, timeout ti
 	configs := make([]*resource.StateChangeConf, 0, len(changes))
 	for change, changeValue := range changes {
 		switch change {
-		case networkDeviceSchemaNames["ACLTemplateUUID"]:
+		case networkDeviceSchemaNames["ACLTemplateUUID"], networkDeviceSchemaNames["MgmtAclTemplateUuid"]:
 			aclTempID, ok := changeValue.(string)
 			if !ok || aclTempID == "" {
 				break
```

Comment:
Should this be `continue`?

Reply:
For example, what would happen if the user had:
```hcl
resource "equinix_network_device" "foo" {
  acl_template_uuid = "<uuid>"
  mgmt_acl_template_uuid = "<uuid>"
  ...
}
```
and then changed to:
```
resource "equinix_network_device" "foo" {
  mgmt_acl_template_uuid = "<uuid>"
  ...
}
```

I'm wondering if we end up with `aclTempID == ""` and we `break` out of the loop without updating the other changes (including `AdditionalBandwidth`).

Reply:
@rling-equinix you mentioned that https://github.com/equinix/terraform-provider-equinix/blob/63f33e2a542a6e8530c11a41e2f8ffd13a434178/equinix/resource_network_device.go#L961-L986 will always log the warning because the ACL can not be reset to `""`. 

Should we open an issue to track that or address it here?

Reply:
We should open an issue to track that. Maybe in the future, we will support deleting ACL from the device.

Reply:
https://github.com/equinix/terraform-provider-equinix/issues/118